### PR TITLE
Module 3 Fan Page 

### PR DESCRIPTION
**Pull request by andrewmdav**
External CSS files promote the efficient maintenance of web sites, because the formatting of multiple pages can be updated in a single document.  Consistency across a multiple page site is promoted, leaving page-by-page, or even worse, line-by-line formatting behind.

### CODE REVIEW

File: fan-page.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+h1 {
+    color: blanchedalmond
+}
+
+#divcolor {
+    color: teal
+}
+
+.feature {
+    background-color: aqua
```
Comment (mbMosman) [inline fan-page.css]: Be careful between text color and background color requirements.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title> First-Try Robotics</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="fan-page.css">
+    <style>
```
Comment (mbMosman) [inline index.html]: Pull out the embedded style if not used.  You should not be using these outside of specific requirements to do so (such as in CSS Cascade assignment.)

Comment (andrewmdav) [inline index.html]: Makes sense - thanks!

File: fan-page.css (hunk 1 above)
Comment (andrewmdav) [inline fan-page.css]: Yes, I see my confusion.

**Pull request closed without merging**